### DISCUSSION TITLE
Wider scrollbar

### DISCUSSION THREAD

**Discussion by milsap1234** [tsujan/FeatherPad#559] [Q&A] [answered]
Featherpad is my preferred text editor, but unfortunately the vertical scrollbar is narrow and hard to grab.  Is there a way to increase its width?  Thanks.

System:
Desktop: Xfce 4.16.0 tk: Gtk 3.24.29 info: xfce4-panel wm: xfwm4 
Distro: Artix Linux base: Arch Linux 
Style Adwaita-dark (no change in scrollbar with other available styles)

![Screenshot_2021-08-24_04-43-33](https://user-images.githubusercontent.com/57337140/130672490-b3372a3d-106b-407a-9ed0-0a0ae7476c8c.jpg)

**Comment by tsujan** [comment, marked as answer]
The scrollbar width is determined by the active widget style, not by FeatherPad. No app should enforce such sizes; it's the job of Qt widget styles/themes to set them.

This screenshot is taken with Fusion, Qt's default widget style:

![scrollbar](https://user-images.githubusercontent.com/981076/130674497-6d25cf41-19e9-4f04-8837-d6be79bd9807.png)

And this one with a Kvantum theme:

![scrollbar1](https://user-images.githubusercontent.com/981076/130675034-4c369e8e-1fc3-4ded-945d-bb38801238e1.png)

**Reply by milsap1234** [reply]
Thank you.
